<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogCameraSurvey</class>
 <widget class="QDialog" name="DialogCameraSurvey">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>30</y>
     <width>81</width>
     <height>241</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonIntEle">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>170</y>
     <width>41</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>三维控制点文件</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditFeatPt">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>391</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDest">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>230</y>
     <width>41</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditIntEle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>391</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>影像特征点文件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonFeatPoint">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>100</y>
     <width>41</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>外方位元素及精度结果</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditDest">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>391</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogCameraSurvey</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogCameraSurvey</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
